tests/integ/modin/frame/test_describe.py

```diff
@@ -308,26 +308,13 @@ def test_describe_multiindex(index, columns, include, expected_union_count):
         )
 
 
-DUP_COL_FAIL_REASON = "SNOW-1019479: describe on frames with mixed object/number columns with the same name fails"
-
-
 @pytest.mark.parametrize(
     "include, exclude, expected_union_count",
     [
         (None, None, 7),
-        pytest.param(
```

Comment:
do we have a test that one column or more columns is just all None ?

Reply:
A few tests have parameters that have all-None columns in the data (`test_describe_numeric_only`, `test_describe_obj_only`), but I didn't include any specifically for this test. Do you think it's worth adding?

Reply:
i think what i would be really interested is see what happens if a we have a column that have float numbers, but configured as object type, and if a column have mixed float, int and None marked as object what happens there.

Reply:
The columns included in each set of aggregations are determined by the dtype of the column, not the type of the objects themselves. A column of float or mixed numeric/NULL values with `object` dtype would be treated the same as one with only non-numeric ones (top/freq are computed, std/min/max/etc. are not).